apps/studio/src/store/modules/TabModule.ts

```diff
@@ -86,6 +106,13 @@ export const TabModule: Module<State, RootState> = {
         item.connectionId = usedConfig.id
         await item.save()
       }
+
+      // Prevent multiple tabs per table
+      const existingTabInClosedTabs = context.state.lastClosedTabs.find((tab) => tab.matches(item))
+      if (existingTabInClosedTabs) {
```

Comment:
this can just happen in `add`, no need to put it here else we'll forget about it.